docs/manual/en/manual.txt

```diff
@@ -0,0 +1,297 @@
+// -*- mode: adoc; -*-
+
+:lang: en
+
+= User's Guide
+Jean Pierre Cimalando, Olivier Humbert, Bruce Sutherland
+v1.2
+:toc:
+
+[.Lead]
+This manual documents the use of ADLplug, a frequency modulation (FM) synthesizer based on the technology of the OPL3. +
+ADLplug is free and open source software. +
+If you encounter a problem with the software, please create an issue on the project page: +
+https://github.com/jpcima/ADLplug/issues
+
+== Description
+
+ADLplug is an FM synthesizer, built on Yamaha's OPL3 standard.
```

Comment:
@trebmuh a / an before an acronym in English is tricky. In this case, the acronym is generally pronounced "eff-em". Since that starts with a vowel sound, we use 'an', not 'a'.

https://english.stackexchange.com/questions/468475/should-one-use-an-or-an-a-before-an-acronym-beginning-with-the-letter-h

Reply:
ok

Reply:
Thank you for the explanation.